expenses-manager-frontend/src/main/frontend/src/app/sidebar/sidebar.component.scss

```diff
@@ -0,0 +1,43 @@
+.sidebar{
+    width: 220px;
+    height: 100%;
+    background-color: #4F46E5;
```

Comment:
Aceeasi observatie pe care am lasat-o aici: https://github.com/EndavaCollab/bvdapprenticeship2024-expenses-manager/pull/4
1. Sa folosim rem, inloc de px.
2. Sa folosim inline-size si block-size in loc de width si height.
3. Sa ne definim paleta de culori cu variabile in styles.css iar aici sa o folosim background-color: var(--primary-color);
```
:root {
  --primary-color: #4F46E5;
}
```
E posibil ca aceste fix-uri sa fie disponibile dupa rebase daca PR lui Vlad va fi merge-uit primul.